### PR TITLE
Fix test imports

### PR DESCRIPTION
**Pull request by gpsamson**
- Fixes an issue where tests would fail without a `yarn build`
- Removes the build step when testing the set of setup commands documented for external devs

## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [ ] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [ ] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/destination-actions/tsconfig.json [hunk 1]
```diff
@@ -3,10 +3,20 @@
   "compilerOptions": {
     "baseUrl": ".",
     "paths": {
-      "@segment/actions-core": ["../core/src"],
-      "@segment/destination-subscriptions": ["../destination-subscriptions/src"]
+      "@segment/ajv-human-errors": [
```
Comment (gpsamson) [inline packages/destination-actions/tsconfig.json:6]: need to fix formatting

File: .github/workflows/ext.yml [hunk 2]
```diff
@@ -31,8 +31,8 @@ jobs:
...
         run: yarn bootstrap --no-ci
 
-      - name: Build
-        run: NODE_ENV=production yarn build
-
       - name: Test
         run: yarn test
+
+      - name: Build
+        run: yarn build
```
Comment (mshwery) [inline .github/workflows/ext.yml:38]: Should this be `NODE_ENV=production`? or is that not needed?

File: packages/core/tsconfig.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "./tsconfig.build.json",
   "compilerOptions": {
+    "baseUrl": "./",
```
Comment (mshwery) [inline packages/core/tsconfig.json:4]: Is this needed?

Comment (mshwery) [reply packages/core/tsconfig.json:4]: I've found baseUrl and rootDir changes to introduce somewhat unpredictable breakages, so I'm generally hesitant.

**Review by mshwery — APPROVED** [review]

File: packages/core/tsconfig.json (hunk 3 above)
Comment (gpsamson) [reply packages/core/tsconfig.json:4]: I think `baseUrl` is needed if `paths` is used..

File: .github/workflows/ext.yml (hunk 2 above)
Comment (gpsamson) [reply .github/workflows/ext.yml:38]: I removed it from the external workflow to better emulate the dev workflow. I don't think devs will specify production when running locally.

**Pull request merged**